erpnext/selling/doctype/customer/customer.py

```diff
@@ -326,16 +327,39 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
 		credit_controller_role = frappe.db.get_single_value('Accounts Settings', 'credit_controller')
 		if not credit_controller_role or credit_controller_role not in frappe.get_roles():
 			credit_controller_users = get_users_with_role(credit_controller_role or "Sales Master Manager")
+			credit_controller_users_list = [user for user in credit_controller_users
+				if frappe.db.exists("Employee", {"prefered_email": user})]
 			credit_controller_users = [get_formatted_email(user) for user in credit_controller_users
 				if frappe.db.exists("Employee", {"prefered_email": user})]
```

Comment:
If you're doing the above addition, change this to:

```suggestion
			credit_controller_users = [get_formatted_email(user) for user in credit_controller_users_list]
```